microsoft/botframework-solutions
# [Virtual Assistant Template][TypeScript] Update template behavior

## Description

- Update the behavior of `mainDialog`
- Remove `authenticationDialog` and `dialogBase`
- Remove `authenticationResponses`
- Add `templateManager` folder 
- Update `locales`

Related to #1058 

## Testing Steps
Using the `generator-botbuilder-assistant`
1. Open a CLI and run `yo botbuilder-assistant` wherever you want to create an assistant
1. Follow the prompts to create one
1. Check that the assistant was created successfully
1. Deploy a bot running `deploy.ps1` file inside of `deployment/scripts` folder
1. Run `npm run start`
1. Check that the assistant has the same behavior to C# version

## Checklist
